internal/gojs/testdata/writefs/main.go

```diff
@@ -0,0 +1,46 @@
+package writefs
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+)
+
+func Main() {
```

Comment:
@achille-roussel since I already had work done and mind focused on GOOS=js, I spiked this. We'll likely need to have multiple pre-opens in order to properly address folks who want to map /tmp to a different FS.

To complete WASI, we need a similar sort of testdata in that directory, run with `wasi_stdlib_test.go`